ToDoLine/Properties/launchSettings.json

```diff
@@ -2,10 +2,12 @@
   "profiles": {
     "ToDoLine": {
       "commandName": "Project",
-      "launchBrowser": false,
+      "launchBrowser": true,
+      "launchUrl": "odata/swagger", 
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "applicationUrl": "http://localhost:53200/"
```

Comment:
It doesn't support wildcard for host name. See https://github.com/bit-foundation/ToDoLine/blob/master/ToDoLine/Program.cs#L19 Remove application url, i'll accept launchBrowser and launchUrl changes then.